pos_ticket_without_price/static/src/xml/pos.xml

```diff
@@ -30,4 +30,35 @@
         </t>
     </t>
 
+    <t t-name="XmlReceiptWithoutPrice" t-extend="XmlReceipt">
+        <t t-jquery="t[t-if='receipt.company.logo']" t-operation="before">
+            <h1>TICKET WITHOUT PRICE</h1>  
+        </t>
+        
+        <t t-jquery="t[t-foreach='receipt.orderlines']" t-operation="replace">
+            <t t-foreach='receipt.orderlines' t-as='line'>
+                 <value t-att-value-decimals='pos.dp["Product Unit of Measure"]' value-autoint='on'>
+                    <t t-esc='line.quantity' />
```

Comment:
Have you consireded adding `line.unit_name`?

Reply:
i try to put less information but i put  `<t t-esc='line.product_name_wrapped[0]' />`
What its exactly the line.unit_name content?

Reply:
You can check `line.unit_name` usage in original `XmlReceipt`: https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/static/src/xml/pos.xml#L798

Currently that value (only the unit description if this is not "Unit(s)") is rendered; since it's not a price value, the value should remain within the receipt IMHO.